requirements.txt

```diff
@@ -5,7 +5,7 @@ Django==4.2.13
 dj-database-url==2.2.0
 django-allauth[socialaccount]==0.63.3
 django-cors-headers==4.3.1
-django-csp==3.8
+django-csp==4.0b1
```

Comment:
question (non-blocking): We don't seem to use any other beta versions of any libraries. When this is out of beta, I assume we'll be able to update this from `4.0b1` to just `4.0`?

Reply:
I assume so too, but that's a Dependabot question